src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
+        <input id="cellsynt-login" v-model="$parent.notification.cellsyntLogin" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-key" class="form-label">{{ $t("Password") }}</label>
+        <HiddenInput id="cellsynt-key" v-model="$parent.notification.cellsyntPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-Originatortype" class="form-label">{{ $t("Originator type") }}</label>
+        <select id="cellsynt-Originatortype" v-model="$parent.notification.cellsyntOriginatortype" :required="true" class="form-select">
+            <option value="alpha">{{ $t("Alpha (recommended)") }}</option>
+            <option value="numeric">{{ $t("Numeric") }}</option>
+        </select>
+        <div class="form-text">
+            <p><b>{{ $t("Alpha (recommended)") }}:</b><br /> {{ $t("Alphanumeric string (max 11 alphanumeric characters). Recipients can not reply to the message.") }}</p>
+            <p><b>{{ $t("Numeric") }}:</b><br /> {{ $t("Numeric value (max 15 digits) with telephone number on international format without leading 00 (example UK number 07920 110 000 should be set as 447920110000). Recipients can reply to the message.") }}</p>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-originator" class="form-label">{{ $t("Originator") }}</label>
+        <input id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="text" class="form-control" required>
```

Comment:
https://github.com/louislam/uptime-kuma/pull/4558#discussion_r1538410925

Reply:
I have added pattern, max-length and change with v-if depending on what the user selects as originator type.